plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/AppEngineTabGroup.java

```diff
@@ -21,24 +21,38 @@
 import org.eclipse.debug.ui.EnvironmentTab;
 import org.eclipse.debug.ui.ILaunchConfigurationDialog;
 import org.eclipse.debug.ui.ILaunchConfigurationTab;
+import org.eclipse.debug.ui.sourcelookup.SourceLookupTab;
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.wst.server.ui.ServerLaunchConfigurationTab;
+import org.eclipse.jdt.debug.ui.launchConfigurations.JavaArgumentsTab;
+import org.eclipse.jdt.debug.ui.launchConfigurations.JavaClasspathTab;
 
+/**
+ *  Tabs shown in launch configurations for an App Engine Server.
+ *  To get to these in the UI:
+ *  
+ * 1. Open Servers view
+ * 2. Right click on the server to configure
+ * 3. Open
+ * 4. Open Launch Configuration
```

Comment:
FYI, there are [other ways to get to these](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/467#issuecomment-267639717).